deploy/scripts/src/txma/test.ts

```diff
@@ -46,9 +46,9 @@ const profiles: ProfileList = {
       startRate: 1,
       timeUnit: '1s',
       preAllocatedVUs: 1,
-      maxVUs: 700,
+      maxVUs: 1000,
       stages: [
-        { target: 20, duration: '10m' } // Ramp up to 20 iterations per second in 10 minutes
+        { target: 100, duration: '10m' } // Ramp up to 20 iterations per second in 10 minutes
```

Comment:
```suggestion
        { target: 100, duration: '10m' } // Ramp up to 100 iterations per second in 10 minutes
```
Update comments to match values

Reply:
Done